guacamole-common-js/src/main/webapp/modules/KeyEventInterpreter.js

```diff
@@ -283,7 +179,8 @@ Guacamole.KeyEventInterpreter = function KeyEventInterpreter(batchSeperation, st
         var name = String.fromCharCode(codepoint);
 
         // Create and return the definition
-        return new KeyDefinition({keysym: keysym, name: name, value: name, modifier: false});
+        return new Guacamole.KeyEventInterpreter.KeyDefinition({
+                keysym: keysym, name: name, value: name);
```

Comment:
```
#17 31.59 [INFO] BUILD FAILURE
#17 31.59 [INFO] ------------------------------------------------------------------------
#17 31.59 [INFO] Total time:  30.193 s
#17 31.59 [INFO] Finished at: 2023-09-27T17:49:53Z
#17 31.59 [INFO] ------------------------------------------------------------------------
#17 31.59 [ERROR] Failed to execute goal com.github.buckelieg:minify-maven-plugin:2.0.1:minify (default-minify) on project guacamole-common-js: java.lang.RuntimeException: JSCompiler errors
#17 31.59 [ERROR] all.js:8606: ERROR - [JSC_PARSE_ERROR] Parse error. '}' expected
#17 31.59 [ERROR]                 keysym: keysym, name: name, value: name);
#17 31.59 [ERROR]                                                        ^
#17 31.59 [ERROR] 
#17 31.59 [ERROR] -> [Help 1]
```

Reply:
Oops, sorry. I think my IDE ate that closing brace. Should be fixed now.

Reply:
😋